images/driver-toolkit.yml

```diff
@@ -26,9 +26,6 @@ enabled_repos:
 - rhel-9-server-ose-rpms-embargoed
 - rhel-9-baseos-rpms
 - rhel-9-appstream-rpms
-- rhel-92-baseos-rpms
-- rhel-92-appstream-rpms
-- rhel-92-rt-rpms
```

Comment:
actually we still need the rt-rpms repo. the build works without it because we've shipped `kernel-rt` ourselves but that will fail when RHEL RT gets ahead of us. we missed this in 4.14 too.

Reply:
https://github.com/openshift-eng/ocp-build-data/pull/2966